graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationDirectives.java

```diff
@@ -1,20 +1,14 @@
 package com.apollographql.federation.graphqljava;
 
-import static graphql.introspection.Introspection.DirectiveLocation.FIELD_DEFINITION;
-import static graphql.introspection.Introspection.DirectiveLocation.INTERFACE;
-import static graphql.introspection.Introspection.DirectiveLocation.OBJECT;
+import static graphql.introspection.Introspection.DirectiveLocation.*;
 import static graphql.language.DirectiveDefinition.newDirectiveDefinition;
 import static graphql.language.DirectiveLocation.newDirectiveLocation;
 import static graphql.language.InputValueDefinition.newInputValueDefinition;
 import static graphql.schema.GraphQLArgument.newArgument;
 import static graphql.schema.GraphQLDirective.newDirective;
 
 import graphql.PublicApi;
-import graphql.language.DirectiveDefinition;
-import graphql.language.DirectiveLocation;
-import graphql.language.InputValueDefinition;
-import graphql.language.NonNullType;
-import graphql.language.TypeName;
+import graphql.language.*;
```

Comment:
My intelliJ did that. Are we OK with star imports? If not we should certainly commit some project configuration somewhere so we have all the same.